### PR TITLE
remove conceptual

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@danroth27 please review

**Comment by Rick-Anderson** [comment]
fixes #512 sans the email.

File: aspnet/fundamentals/index.rst [hunk 1]
```diff
@@ -1,10 +1,11 @@
-Fundamentals
-============
+Fundamentals z
```
Comment (danroth27) [inline aspnet/fundamentals/index.rst]: ???

File: aspnet/fundamentals/index.rst [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+===============
 
 .. toctree::
     :titlesonly:
 
     startup
+    choosing-the-right-dotnet
```
Comment (danroth27) [inline aspnet/fundamentals/index.rst]: Let's put this at the bottom of Fundamentals. My gut is that this article eventually needs to move out of the ASP.NET Core docs to be a general .NET article.

File: aspnet/fundamentals/static-files.rst [hunk 3]
```diff
@@ -15,7 +15,7 @@ Static files, such as HTML, CSS, image, and JavaScript, are assets that an ASP.N
 Serving static files
```
Comment (danroth27) [inline aspnet/fundamentals/static-files.rst]: Why is this file in this PR?

**Comment by danroth27** [comment]
A coupe of minor nits :watch:

File: aspnet/index.rst [hunk 4]
```diff
@@ -18,10 +18,9 @@ Topics
 .. toctree::
     :titlesonly:
 
-    getting-started
-    API <https://docs.asp.net/projects/api>
+    intro
```
Comment (danroth27) [inline aspnet/index.rst]: Change this to `Introduction <intro>`

**Comment by danroth27** [comment]
:watch:

**Comment by danroth27** [comment]
:shipit:

**Pull request merged**